lib/image-diff.js

```diff
@@ -8,25 +9,25 @@ var mkdirp = require('mkdirp');
 var tmp = require('tmp');
 
 // Define custom resize function
-// Taken from https://github.com/twolfson/twolfson.com/blob/3.4.0/test/perceptual-tests/twolfson.com_test.js#L88-L107
-// TODO: Make image resizing its own library
-// DEV: This does not pollute gm.prototype
-gm.prototype.fillFromTo = function (params) {
+function transparentExtent(gm, params) {
+  // Assert we received our parameters
+  assert.notEqual(params.width, undefined);
+  assert.notEqual(params.height, undefined);
+
   // Fill in new space with white background
   // TODO: Parameterize background color (should be considered 'transparent' color in this case)
-  this.borderColor('transparent');
-  this.border(Math.max(params.toWidth - params.fromWidth, 0), Math.max(params.toHeight - params.fromHeight, 0));
+  gm.background('transparent');
 
   // Anchor image to upper-left
   // TODO: Parameterize anchor point
-  this.gravity('SouthEast');
+  gm.gravity('NorthWest');
```

Comment:
Yea, it's the default but I prefer to be explicit and we could parameterize it easily. For example, in the following images, what if someone wanted the smaller one to move to the lower right when upscaled?

https://github.com/uber/image-diff/blob/1.3.0/test/test-files/checkerboard.png

https://github.com/uber/image-diff/blob/1.3.0/test/test-files/checkerboard-excess.png

Note: Currently for gravity, it will make the 2 images look the same except there will be a transparent edge for the smaller one